app/src/marketplace/containers/EditProductPage/ProductStreams.jsx

```diff
@@ -46,16 +46,12 @@ const ProductStreams = ({ disabled }: Props) => {
     return (
         <section id="streams" className={cx(styles.root, styles.StreamSelector)}>
             <div>
-                <Translate
-                    tag="h1"
-                    value="editProductPage.streams.title"
-                />
-                <Translate
-                    tag="p"
-                    value="editProductPage.streams.description"
-                    docsLink={docsLinks.createProduct}
-                    dangerousHTML
-                />
+                <h1>Add streams</h1>
+                <p>
+                    Products can contain a range of streams, or a single &quot;firehose&quot; type stream, it&#39;s up to you.
+                    If you haven&#39;t made any streams yet, you can create one here. For help creating streams,
```

Comment:
`&apos;` is pretty much `&#39;` but I find the first one easier to read as a human. ;)